src/Protocol/Protocol_1_13.cpp

```diff
@@ -69,6 +72,25 @@ cProtocol_1_13::cProtocol_1_13(cClientHandle * a_Client, const AString & a_Serve
 
 
 
+void cProtocol_1_13::Initialize(cClientHandle & a_Client)
+{
+	// Get the palettes; fail if not available:
+	auto paletteVersion = this->GetPaletteVersion();
+	m_BlockTypePalette = cRoot::Get()->GetProtocolPalettes().blockTypePalette(paletteVersion);
+	if (m_BlockTypePalette == nullptr)
+	{
+		throw std::runtime_error(Printf("This server doesn't support protocol %s.", paletteVersion));
+	}
+
+	// Process the palette into the temporary BLOCKTYPE -> NetBlockID map:
+	auto upg = cRoot::Get()->GetUpgradeBlockTypePalette();
+	m_BlockTypeMap = m_BlockTypePalette->createTransformMapWithFallback(upg, 0);
```

Comment:
For now the `m_BlockTypeMap` is used for the mapping while serializing chunks; when block states are implemented in Cuberite, `m_BlockTypePalette` will be used for the serialization.